src/commands/contract/download_wasm/mod.rs

```diff
@@ -50,6 +100,9 @@ impl Contract {
 #[interactive_clap(input_context = ContractContext)]
 #[interactive_clap(output_context = DownloadContractContext)]
 pub struct DownloadContract {
```

Comment:
So you are modifying `download_wasm`. Is it possible to support `download_abi` as well as other methods? Are they working right now for global contracts?
Anyway, maybe that's for a different PR.